lib/Commands/NinjaBuildCommand.cpp

```diff
@@ -146,7 +146,7 @@ struct BuildExecutionQueue {
   bool useLIFO;
 
 public:
-  BuildExecutionQueue(unsigned numLanes, bool useLIFO)
+  BuildExecutionQueue(unsigned long numLanes, bool useLIFO)
```

Comment:
I would rather not use `long` in interfaces... I think it is rarely the right type. If a larger type than int is needed then `uint64_t` is my preferred type (which is not platform dependent). In this case, though, the extra size is never needed here.